src/omotes_orchestrator/db_models/job_starts.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+import sqlalchemy as db
+from sqlalchemy import ForeignKey
+
+from omotes_orchestrator.db_models.base import Base
+
+
+@dataclass
+class JobStartsDB(Base):
```

Comment:
In our case we want to be able to count how many times a job has been started. With a single value e.g. `started_at` we can only see when the latest job was started. I also thought about adding a counter column in `JobDB` but then we wouldn't know when the previous jobs were started. Sometimes this info is quite useful when debugging so I settled on adding a new table. In case we want to add more info (e.g. worker id that was processing the job) later, it would be possible with this pattern.

Reply:
Clear, thanks!